docusaurus/docs/operate/quickstart/supplier_cheatsheet.md

```diff
@@ -0,0 +1,265 @@
+- [Supplier FAQ](#supplier-faq)
+  - [What Supplier transactions are available?](#what-supplier-transactions-are-available)
+  - [What Supplier queries are available?](#what-supplier-queries-are-available)
+  - [How do I query for all existing onchain Suppliers?](#how-do-i-query-for-all-existing-onchain-suppliers)
+
+:::note
+
+For detailed instructions, troubleshooting, and observability setup, see the [Supplier Walkthrough](./../run_a_node/supplier_walkthrough.md).
+
+:::
+
+## Pre-Requisites
+
+1. Make sure to [install the `poktrolld` CLI](../user_guide/install.md).
+2. Make sure you know how to [create and fund a new account](../user_guide/create-new-wallet.md).
+3. You have either [staked a new `service` or found an existing one](./service_cheatsheet.md).
+4. `[Optional]` You can run things locally or have dedicated long-running hardware. See the [Docker Compose Cheat Sheet](./docker_compose_debian_cheatsheet#deploy-your-server) if you're interested in the latter.
+
+### Context
+
+This document is a cheat sheet to get you quickly started with two things:
+
+1. Staking an onchain `Supplier`
+2. Deploying an offchain `RelayMiner`
+
+By the end of it, you should be able to serve Relays onchain.
+
+## Account Setup
+
+### Create and fund the `Supplier` account
+
+Create a new key pair for the `Supplier`
+
+```bash
+poktrolld keys add supplier
+
+# Optionally, to avoid entering the password each time:
+# poktrolld keys add supplier --keyring-backend test
+```
+
+:::tip
+
+You can set the `--keyring-backend` flag to `test` to avoid entering the password
+each time.
+
+Learn more about [cosmos keyring backends here](https://docs.cosmos.network/v0.46/run-node/keyring.html).
+
+:::
+
+### Prepare your environment
+
+For convenience, we're setting several environment variables to streamline
+the process of interacting with the Shannon network:
+
+We recommend you put these in your `~/.bashrc` file:
+
+```bash
+export NODE="https://shannon-testnet-grove-rpc.beta.poktroll.com"
+export NODE_FLAGS="--node=https://shannon-testnet-grove-rpc.beta.poktroll.com"
+export TX_PARAM_FLAGS="--gas=auto --gas-prices=1upokt --gas-adjustment=1.5 --chain-id=pocket-beta --yes"
+export SUPPLIER_ADDR=$(poktrolld keys show supplier -a)
+
+# Optionally, to avoid entering the password each time:
+# export SUPPLIER_ADDR=$(poktrolld keys show supplier -a --keyring-backend test
+```
+
+:::tip
+
+You can put the above in a special `~/.poktrollrc` and add `source ~/.poktrollrc` to
+your `~/.profile` file for a cleaner organization.
+
+:::
+
+## Supplier Configuration
+
+### Fund the Supplier account
+
+Run the following command to get the `Supplier`:
+
+```bash
+echo "Supplier address: $SUPPLIER_ADDR"
+```
+
+Then use the [Shannon Beta TestNet faucet](https://faucet.beta.testnet.pokt.network/) to fund the account.
+
+Afterwards, you can query the balance using the following command:
+
+```bash
+poktrolld query bank balances $SUPPLIER_ADDR $NODE_FLAGS
```

Comment:
```suggestion
poktrolld query bank balances $SUPPLIER_ADDR
```

Reply:
Resolving as wont-do and delegating to #1011